spec/index.bs

```diff
@@ -514,16 +550,20 @@ interface GPU {
     The user agent chooses whether to return an adapter, and, if so,
     chooses according to the provided |options|.
 
-    Returns [=a new promise=], |promise|.
+    Returns [=a new promise=], |promise|. Then, the following occurs asynchronously:
```

Comment:
I didn't do this because I wasn't sure what timeline to use. The obvious one is the device timeline, but there's no device yet so it doesn't exactly make sense. Should we rename the device timeline ("service timeline")? add another timeline?

Reply:
We could add some words about adapter and device initialization also happening at "Device" timeline, in a follow-up.